Added ubuntu build script for dc-js

jotirlingSwami opened the pull request:
## Checklist
<!--- Goto Preview tab for better readability -->
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [ ] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

sumitd2 commented:
+ npm install grunt --save-dev
npm ERR! Tracker "idealTree" already exists

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2022-02-11T20_41_10_010Z-debug-0.log

jotirlingSwami commented:
> * npm install grunt --save-dev
>   npm ERR! Tracker "idealTree" already exists
> 
> npm ERR! A complete log of this run can be found in: npm ERR! /root/.npm/_logs/2022-02-11T20_41_10_010Z-debug-0.log

On fresh container it will build without error.
[dc-js build Log.log](https://github.com/ppc64le/build-scripts/files/8057704/dc-js.build.Log.log)

Please find the build log file attached below.

sumitd2 commented:
> > * npm install grunt --save-dev
> >   npm ERR! Tracker "idealTree" already exists
> > 
> > npm ERR! A complete log of this run can be found in: npm ERR! /root/.npm/_logs/2022-02-11T20_41_10_010Z-debug-0.log
> 
> On fresh container it will build without error. Please find the build log file attached below. [dc-js build Log.log](https://github.com/ppc64le/build-scripts/files/8057704/dc-js.build.Log.log)

This error is thrown when the script is run from /

jotirlingSwami commented:
> > > * npm install grunt --save-dev
> > >   npm ERR! Tracker "idealTree" already exists
> > > 
> > > npm ERR! A complete log of this run can be found in: npm ERR! /root/.npm/_logs/2022-02-11T20_41_10_010Z-debug-0.log
> > 
> > 
> > On fresh container it will build without error. Please find the build log file attached below. [dc-js build Log.log](https://github.com/ppc64le/build-scripts/files/8057704/dc-js.build.Log.log)
> 
> This error is thrown when the script is run from /

Addressed the review comments Please review once @sumitd2

seth-priya commented:
looks like there is a conflict now, @jotirlingSwami please check if this PR is still relevant, else close
